lib/helpers.ts

```diff
@@ -0,0 +1,10 @@
+type TenderlyEnvVariables = {
+  TENDERLY_ACCESS_KEY: string;
+  TENDERLY_ACCOUNT_NAME: string;
+  TENDERLY_PROJECT_NAME: string;
+  TENDERLY_GET_BY_PROJECT_NAME: string;
+};
+
+export function getEnvironmentVariables() {
+  return process.env as TenderlyEnvVariables;
+}
```

Comment:
@maleksandar to think about how to organise this

Reply:
for now, here is ok. We don't have many exports from the main path, once we have such an issue, we might introduce namespaces with a major release.